### PR TITLE
Add support for vim-like hjkl movement keys.

### PR DESCRIPTION
**Pull request by stephenwithav**
A simple update to make chorus more comfortable for vim users.

### CODE REVIEW

File: src/js/apps/input/input_app.js.coffee [hunk 1]
```diff
@@ -86,13 +86,13 @@
 
       ## Respond to key code
       switch e.which
-        when 37 # left
+        when 37, 72 # left
```
Comment (SamK) [inline src/js/apps/input/input_app.js.coffee]: You should update the comments as well: `# left, j`

Comment (SamK) [inline src/js/apps/input/input_app.js.coffee]: I'm no Javascript expert but it looks like 72 is upper case H. Lower case h is 104.
Source: https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_event_key_keycode

Comment (stephenwithav) [inline src/js/apps/input/input_app.js.coffee]: re: `72 vs 104`, chorus2 uses the upper case keycodes for `x` and `t`, so I followed that convention.

Good idea re: updating the comments.

**Comment by jez500** [comment]
Thanks @stephenwithav

**Pull request merged**